spec.bs

```diff
@@ -6011,8 +6020,8 @@ frequency capping. It's a [=struct=] with the following [=struct/items=]:
 <dl dfn-for="previous win">
   : <dfn>time</dfn>
   :: A [=moment=]. Approximate time the [=interest group=] won an auction.
-  : <dfn>ad json</dfn>
-  :: A [=string=]. A JSON serialized object corresponding to the ad that won the auction.
+  : <dfn>ad</dfn>
+  :: An [=interest group ad=]. The ad that won the auction.

```

Comment:
let's be explicit that this is just a subset of it (list the fields
```suggestion
  :: An [=interest group ad=]. The ad that won the auction.
    Fields except [=interest group ad/render url=], [=interest group ad/metadata=] and [=interest group ad/ad render ID=] are excluded.
```

Reply:
Done.